pkg/automationconfig/automation_config.go

```diff
@@ -185,6 +186,31 @@ type Auth struct {
 	AutoPwd string `json:"autoPwd,omitempty"`
 }
 
+type Prometheus struct {
+	Enabled          bool   `json:"enabled"`
+	Username         string `json:"username"`
+	Password         string `json:"password"`
+	Scheme           string `json:"scheme"`
+	TLSPemPath       string `json:"tlsPemPath"`
+	TLSPemPassword   string `json:"tlsPemPassword"`
+	Mode             string `json:"mode"`
+	ListenAddress    string `json:"listenAddress"`
+	MetricsPath      string `json:"metricsPath"`
+	ServiceDiscovery string `json:"serviceDiscovery"`
+}
+
+func NewDefaultPrometheus(username string) Prometheus {
+	return Prometheus{
+		Enabled:          true,
+		Username:         username,
+		Scheme:           "http",
+		Mode:             "opsManager",
+		ListenAddress:    "0.0.0.0:9216",
```

Comment:
Should we hardcode the port here? can it be changed on some usecase?

Reply:
It can be changed via `prometheus.port`